addons/binding/org.openhab.binding.elkm1/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.elkm1
+Bundle-ManifestVersion: 2
+Bundle-Name: Elk M1 Binding
+Bundle-SymbolicName: org.openhab.binding.elkm1;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.5.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.collect,
```

Comment:
Please remove the `com.google.common.collect` import and use the Java 8 constructs instead (e.g. `Stream.of(..).collect(..)`).